zerver/models/recipients.py

```diff
@@ -165,7 +167,8 @@ def get_or_create_direct_message_group(id_list: list[int]) -> DirectMessageGroup
     direct_message_group_hash = get_direct_message_group_hash(id_list)
     with transaction.atomic():
         (direct_message_group, created) = DirectMessageGroup.objects.get_or_create(
-            huddle_hash=direct_message_group_hash
+            huddle_hash=direct_message_group_hash,
+            group_size=len(id_list),
```

Comment:
This looks correct, but probably you should add some assertions to tests that interact with this table to make sure that they have correct values. `TestBulkGetDirectMessageGroupUserIds` seems like a good place to tweak.

Reply:
Done!